lib/rules/test-case-property-ordering.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @fileoverview Requires the properties of a test case to be placed in the given order.
+ * @author 薛定谔的猫<hh_2013@foxmail.com>
+ */
+
+'use strict';
+
+const utils = require('../utils');
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Requires the properties of a test case to be placed in the given order',
+      category: 'Tests',
+      recommended: false,
+    },
+    fixable: null,  // or "code" or "whitespace"
+    schema: [{
+      type: 'array',
+      elements: { type: 'string' },
+    }],
+  },
+
+  create (context) {
+    // ----------------------------------------------------------------------
+    // Public
+    // ----------------------------------------------------------------------
+    const message = 'The properties of a test case should be placed in the given order: [{{order}}].';
```

Comment:
Can we change the error message to only contain properties that appear in the test case? For example:

```js
{ output: "baz", code: "foo" }
// -> The properties of a test case should be placed in the given order: [code, output]

{ code: "foo", errors: ["bar"], output: "baz" }
// -> The properties of a test case should be placed in the given order: [code, output, errors]
```